packages/server/src/dto/DocumentStoreDTO.ts

```diff
@@ -0,0 +1,77 @@
+import { DocumentStoreStatus } from '../Interface'
```

Comment:
what does `dto` stands for?

Reply:
[DTO Pattern](https://dev.to/tamj0rd2/dto-a-typescript-utility-type-4o3m).

Using a DTO as the entity as a number of stringified JSON objects, the front end will not need to see the underlying entity structure